docs_src/README.md

```diff
@@ -4,7 +4,11 @@
 
 # **XLS**: Accelerated HW Synthesis
 
-<!-- GitHub banner -->
+[**Docs**](https://google.github.io/xls/) | [**Quick Start**](https://google.github.io/xls/tools_quick_start/) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://bit.ly/learn-xls) | [**Tutorials**](https://google.github.io/xls/tutorials/)
```

Comment:
this github banner stuff should stay as before as it is auto-generated.

Reply:
done.